Tests/Fixtures/external_target_test/test_project.yml

```diff
@@ -0,0 +1,5 @@
+name: TestProject
+targets:
+  ExternalTarget:
+    type: framework
+    platform: iOS
```

Comment:
We now have Tests/Fixtures/TestProject/AnotherProject. Can we add a target to that instead, instead of creating a new one

Reply:
Also once that's done, the TestProject/project.yml should include a target from that project, as an integration test